### ISSUE TITLE
Upgrading JDK version from 11 to 20

### ISSUE THREAD

**Issue by SaptarshiSarkar12** [google/copybara#244]
The JDK version used in this project is **11** which has been superseded by the latest version **JDK 20**. Many features and fixes are included with JDK 20. So, I think it will be better to **upgrade JDK from 11 to 20**.

**Comment by hsudhof** [comment]
We plan to update once the next LTS release (21) of the JDK lands and is supported by all internal environments.

**Comment by SaptarshiSarkar12** [comment]
> We plan to update once the next LTS release (21) of the JDK lands and is supported by all internal environments.

Ok, that's cool @hsudhof! May I work on that, once JDK 21 gets released?

**Comment by abitrolly** [comment]
JDK 21 is released https://blogs.oracle.com/java/post/the-arrival-of-java-21

**Comment by SaptarshiSarkar12** [comment]
Yeah, right @abitrolly.
@hsudhof Can I work on this issue?
